README.md

```diff
@@ -71,8 +81,8 @@ Now you are running the Krun kernel. Let's check it all looks OK:
    /boot/config-`uname -r` ``. You should see `CONFIG_NO_HZ_FULL_ALL=y` in the
    output.
 
- * Check the syscalls work with `cd krun && make && ./test_prog` (in the
-   top-level of this repo). This runs a program which resets the MSRs and
+ * You can check the syscalls work with `cd krun && make && ./test_prog` (in the
```

Comment:
Is there not some version of this program in the krun repo or some alternate script\program to run instead?

Reply:
There is not. It lives in this repo. To be honest, you can probably skip that step, as I will have done that test before distributing the kernel.